.github/tests/consensus/sovereign.yml

```diff
@@ -1,6 +1,8 @@
 # only supported by fork12 for now
 args:
   consensus_contract_type: pessimistic
+  pp_vkey_hash: "0x00d6e4bdab9cac75a50d58262bb4e60b3107a6b61131ccdff649576c624b6fb7"
+  aggchain_vkey_hash: "0x1f3df68a258f9d6748b291dd35faf80065af7cc11ef8b5fc6db2a4d958ea62d2"
```

Comment:
This might not make sense - we need to differentiate pp with cdk-erigon and pp with op-geth I think

Reply:
Addressed by [446b41c](https://github.com/0xPolygon/kurtosis-cdk/pull/586/commits/446b41c1dd1aca0ada837423fe98b9b291f169c3)